sklearn/feature_selection/tests/test_base.py

```diff
@@ -113,3 +116,37 @@ def test_get_support():
     sel.fit(X, y)
     assert_array_equal(support, sel.get_support())
     assert_array_equal(support_inds, sel.get_support(indices=True))
+
+
+def test_featurewise_scorer():
+    X, y = make_classification(random_state=0)
+
+    # spearmanr from scipy.stats
+    skb = SelectKBest(featurewise_scorer(spearmanr, axis=0), k=10)
```

Comment:
You should really be testing the new function alone. We already have checked that selectkbest works.

Reply:
I meant to test if the wrapper is working as it is supposed to be used. I will try to change the tests, testing the function alone.